docker/deployment/app-deployment.yaml

```diff
@@ -110,8 +110,7 @@ spec:
       initContainers:
       - name: initialize-volume-ownership
         image: busybox:1.33
-        command: ['sh', '-c']
-        args: ['mkdir -p /var/www/html/assets /var/lib/languageforge/lexicon/sendreceive && chown www-data:www-data /var/www/html/assets /var/lib/languageforge/lexicon/sendreceive']
+        command: ['sh', '-c', 'chown www-data:www-data /var/www/html/assets /var/lib/languageforge/lexicon/sendreceive']
```

Comment:
I double-checked the `busybox:1.33` image and it does have UID and GID 33 assigned for the user and group named `www-data`. So doing this will assign UID 33 and GID 33, which is what we want.

Reply:
Thanks for the second set of eyes...this unfortunately is also vulnerable to the underlying image then I take it, e.g., if we went to alpine, would you expect this to break because of the id's being 82 there?